### PR TITLE
[37기 신지윤]ReviewPage 리뷰반영완료

### PR DESCRIPTION
**Pull request by jieyoon22**
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [] 기능 추가
- [x] 리뷰 반영
- [x] 리팩토링
- [x] 버그 수정
- [x] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
수정사항 반영

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
1. 로그인 입력 데이터 유효성 검사 (예시 - 프론트)
사용성없는 콘솔로그 제거
이미지에 alt 추가
클래스명 다이어트
css 클래스명 다이어트
css 컨벤션대로 줄바꿈

<br />

## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)
scss를쓰니 클래스명이 간결해짐을 느꼈습니다

<br />

## :: 기타 질문 및 특이 사항

한가지 문제가 있는데 아직 장현님이 출근을 못하셔서 index,html에 링크를달아놓은게있습니다. 추후엔 사용성이없어 삭제할건데 
삭제후 머지하고 다시추가하는게 옳을까요?

### CODE REVIEW

File: src/Components/ReviewList/ReviewList.js [hunk 1]
```diff
@@ -1,7 +1,86 @@
...
+            <span>
+              <i className="fa-solid fa-arrow-rotate-right" />
+            </span>
+          </span>
+        </span>
+      </div>
+      <div className="reviewListMain">
+        <span className="reviewListMainLeft">
+          <div className="reviewListMainLeftPhoto">
+            <img
```
Comment (ghost) [inline src/Components/ReviewList/ReviewList.js]: __[ESLint]__ img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156283))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `jsx-a11y/alt-text` | `warn` | - |

References:
- <https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156283) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/Components/ReviewList/ReviewList.js [hunk 2]
```diff
@@ -1,7 +1,86 @@
...
+          <div className="reviewListMainCenterPhoto">
+            <GreenMelon />
+            <GreenMelon />
+            <GreenMelon />
+            <GreenMelon />
+            <GreenMelon />
+          </div>
+        </span>
+        <span className="reviewListMainRight">
+          <img
```
Comment (ghost) [inline src/Components/ReviewList/ReviewList.js]: __[ESLint]__ img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156284))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `jsx-a11y/alt-text` | `warn` | - |

References:
- <https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156284) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/Pages/ReviewWrite/GrayMelon.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const GrayMelon = () => {
+  return (
+    <img
```
Comment (ghost) [inline src/Pages/ReviewWrite/GrayMelon.js]: __[ESLint]__ img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156285))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `jsx-a11y/alt-text` | `warn` | - |

References:
- <https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156285) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/Pages/ReviewWrite/ReviewWrite.js [hunk 4]
```diff
@@ -1,7 +1,117 @@
...
+                id="file"
+                multiple="multiple"
+                className="ReviewWritePhotoInnerInput"
+                onChange={saveImages}
+              />
+            </div>
+            <ul className="ReviewWritePhotoInnerPhotos">
+              {images?.map((e, i) => (
+                <li key={i}>
+                  <img src={e} />
```
Comment (ghost) [inline src/Pages/ReviewWrite/ReviewWrite.js]: __[ESLint]__ img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156287))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `jsx-a11y/alt-text` | `warn` | - |

References:
- <https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156287) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/Pages/ReviewWrite/ReviewWrite.js [hunk 5]
```diff
@@ -1,7 +1,117 @@
...
-  return <div>reslissdfsdfsdfsdfst</div>;
+  const [input, setInput] = useState(10);
+  const [filed, setFiled] = useState([]);
+  const [images, setImages] = useState([]);
+  const [save, setSave] = useState([]);
+
+  const saveImages = e => {
+    const fileArr = e.target.files;
+    setSave([...fileArr]);
+    console.log([...save, ...fileArr]);
```
Comment (ghost) [inline src/Pages/ReviewWrite/ReviewWrite.js]: __[ESLint]__ Unexpected console statement. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156288))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-console` | `warn` | No |

References:
- <https://eslint.org/docs/rules/no-console>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156288) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/Pages/ReviewWrite/ReviewWrite.js [hunk 6]
```diff
@@ -1,7 +1,117 @@
-import React from 'react';
+import React, { useState } from 'react';
+import GrayMelon from './GrayMelon';
+import GreenMelon from './GreenMelon';
+import './ReviewWrite.scss';
 
 const ReviewWrite = () => {
-  return <div>reslissdfsdfsdfsdfst</div>;
+  const [input, setInput] = useState(10);
+  const [filed, setFiled] = useState([]);
```
Comment (ghost) [inline src/Pages/ReviewWrite/ReviewWrite.js]: __[ESLint]__ 'setFiled' is assigned a value but never used. ([view](https://sider.review/gh/repos/538372097/pulls/3#issue-8156289))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/538372097/pulls/3#issue-8156289) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

**Review by sstaar91 — COMMENTED** [review]
현재 페이지 아직 진행 중인 것 실제로 확인했으니
이 부분 모두 완료 되면 PR 올려주세요!

**Review by sstaar91 — APPROVED** [review]
수고 많으셨습니다 지윤님!

**Pull request merged**